pkg/prm/status.go

```diff
@@ -0,0 +1,44 @@
+package prm
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/Masterminds/semver"
+)
+
+type Status struct {
+	PuppetVersion *semver.Version
+	Backend       BackendType
+	BackendStatus BackendStatus
+}
+
+func (p *Prm) GetStatus() (status Status) {
+	// Need to mock calls to prm.LoadConfig() so we can set RunningConfig?
+	status.PuppetVersion = RunningConfig.PuppetVersion
+	status.Backend = RunningConfig.Backend
+	status.BackendStatus = p.Backend.Status()
+
+	return status
+}
```

Comment:
Need to see if I can figure out a sensible way to mock `RunningConfig` so this can be appropriately unit tested.